continuationsPlugin/src/main/scala/continuations/Suspend.scala

```diff
@@ -1,9 +1,22 @@
 package continuations
 
+import continuations.Suspend.CompilerRewriteUnsuccessfulException
+
 sealed trait Suspend:
   def continuation[A](f: Continuation[A] => Unit): A
 
+  extension (s: Suspend)
+    inline def suspendContinuation[A](f: Continuation[A] => Unit): A =
+      throw CompilerRewriteUnsuccessfulException
+
 object Suspend:
+  case object CompilerRewriteUnsuccessfulException
+      extends Exception(
+        """|
+           |  The body of this method should have been generated by the continuations-plugin.
+           |  Please open an issue so we can fix this:
+           |  https://github.com/47deg/TBD/issues/new?assignees=&labels=bug&template=compilerrewriteunsuccessfulexception-bug-report.md&title=CompilerRewriteUnsuccessfulException+Bug""".stripMargin)
```

Comment:
the link probably needs update but it will point to this one https://github.com/47deg/TBD/pull/56

Reply:
I approved #56. We should merge that and update the link.

Reply:
ha it actually looks like the link is correct. I hand wrote it so I was expecting to have missed something. but apparently it is good.